src/lspExtensions.ts

```diff
@@ -1,6 +1,19 @@
 import { RequestType } from "vscode-jsonrpc";
 import { TextDocumentIdentifier } from "vscode-languageclient";
+import { LanguageClient } from "vscode-languageclient/node";
 
 export namespace JarClassContentsRequest {
     export const type = new RequestType<TextDocumentIdentifier, string, void>("kotlin/jarClassContents");
 }
+
+export class KotlinApi {
+    private client: LanguageClient;
+
+    constructor(client: LanguageClient) {
+        this.client = client;
+    }
+
+    async getBuildOutputLocation(): Promise<string> {
+        return await this.client.sendRequest("kotlin/buildOutputLocation");
+    }
+}
```

Comment:
Could we define this custom request using a `RequestType` too (for consistency)? Something along the lines of

```suggestion
export namespace JarClassContentsRequest {
    export const type = new RequestType<TextDocumentIdentifier, string, void>("kotlin/jarClassContents");
}

export namespace BuildOutputLocationRequest {
    export const type = new RequestType0<string, void>("kotlin/buildOutputLocation");
}

export class KotlinApi {
    private client: LanguageClient;

    constructor(client: LanguageClient) {
        this.client = client;
    }

    async getBuildOutputLocation(): Promise<string> {
        return await this.client.sendRequest(BuildOutputLocationRequest.type);
    }
}
```